swap_meet/vendor.py

```diff
@@ -1,2 +1,102 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        self.inventory = [] if inventory is None else inventory
+
+    def add(self,item):
+        self.inventory.append(item)
+        return item
+        
+    def remove(self,item):
+        if item not in self.inventory:
+            return False
+        
+        self.inventory.remove(item)
+        return item
+    
+    def get_by_id(self, id):
+        for item in self.inventory:
+            if item.id == id:
+                return item
+        return None
+    
+    def swap_items(self, other_vendor, my_item, their_item):
+        
+        if not my_item in self.inventory or their_item not in other_vendor.inventory:
```

Comment:
We could also use `get_by_id` to check if an item is in a given inventory and then check to see if that function returns `None` or the item.